### PR TITLE
Add workflow for deploying to staging

### PR DESCRIPTION
**Pull request by bion**

### CODE REVIEW

File: .github/workflows/cron.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    runs-on: ubuntu-latest
+
+    env:
+      - AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      - AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - name: check out branch
+        uses: actions/checkout@v2
+
+      - uses: azure/login@v1
```
Comment (shubha-rajan) [inline .github/workflows/cron.yaml:18]: would we need to log in with aws too?

File: .github/workflows/cron.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: resolve latest snapshot tag
+        run: source bin/resolve-latest-snapshot-tag
+
+      - name: Check local dependencies
+        run: bin/doctor "--tag=${LATEST_SNAPSHOT_TAG}"
```
Comment (shubha-rajan) [inline .github/workflows/cron.yaml:26]: what happens if doctor fails?

**Review by shubha-rajan — APPROVED** [review]

File: .github/workflows/cron.yaml (hunk 1 above)
Comment (bion) [reply .github/workflows/cron.yaml:18]: i think AWS CLI picks up the env vars without an explicit call to login

File: .github/workflows/cron.yaml (hunk 2 above)
Comment (bion) [reply .github/workflows/cron.yaml:26]: this is just to help us debug if something isn't working

**Pull request merged**